modules/core/05-port/types/module.go

```diff
@@ -82,4 +82,11 @@ type IBCModule interface {
 		packet channeltypes.Packet,
 		relayer sdk.AccAddress,
 	) error
+
+	NegotiateAppVersion(
```

Comment:
we should add a comment about what the return values means. From my perspective, an error should only be returned if the version negotiation cannot be performed. Maybe this is implicit?

Reply:
Added a short comment above!